analysis/src/lib.rs

```diff
@@ -26,18 +25,6 @@ pub fn analyze<T: FieldElement>(file: ASMProgram<T>) -> Result<AnalysisASMFile<T
     Ok(file)
 }
 
-/// Converts all VMs to constrained machines, replacing
-/// assembly instructions by lookups to programs.
-pub fn convert_vms_to_constrained<T: FieldElement>(file: AnalysisASMFile<T>) -> AnalysisASMFile<T> {
-    // remove all asm (except external instructions)
-    log::debug!("Run asm_to_pil");
-    let file = powdr_asm_to_pil::compile(file);
-
-    // enforce blocks using `operation_id` and `latch`
-    log::debug!("Run enforce_block analysis step");
-    block_enforcer::enforce(file)
```

Comment:
I removed the `block_enforcer::enforce(file)` (now happens as part of `compile`), but then we can just call `powdr_asm_to_pil::compile` directly instead of going through `convert_vms_to_constrained`.